src/components/common/Callout.scss

```diff
@@ -0,0 +1,40 @@
+.Callout {
+  display: flex;
+  position: relative;
+
+  margin-bottom: 24px;
```

Comment:
Don't make this mistake I made in console and regret every day. parent layout belongs in parent components. no padding or you'll have to keep removing and changing it elsewhere. :) remove this :)

Reply:
Point taken.